frontend/src/components/buttons/NavButton.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import { breakpoints } from '../../styled-theme';
+import { Button } from '../base';
+import { ButtonProps } from '../base/Button/Button';
+
+export const NavButton: React.FC<ButtonProps> = ({
+  children,
+  type,
+  onClick,
+  color,
+}) => {
+  return (
+    <NavButtonContainer>
+      <ButtonStyles>
+        <Button type={type} onClick={onClick} color={color}>
+          {children}
+        </Button>
+      </ButtonStyles>
+    </NavButtonContainer>
+  );
+};
+
+export const NavButtonContainer = styled.div`
+  z-index: 30;
```

Comment:
Can you expand on why we need a z-index here?

Reply:
I noticed this was moved around  also; perhaps we don't need it anymore?

Reply:
Z-index is needed for the the `MobileNavItemsContainer` (ie. `background-color: Z-index: 10` and CloseMenuButton `z-index: 20`).  `MobileNavItemsContainer` needs more refactoring.  Specifically, a way to disable x-scrolling.  I've added it to our Notion board.

Reply:
Ahhh okay, makes sense.  Good work man, thanks for the clarification